pulp_rpm/app/serializers.py

```diff
@@ -490,6 +491,9 @@ def create(self, validated_data):
             coll.update_record.add(update_record)
             for package in packages:
                 pkg = UpdateCollectionPackage(**package)
+                if (isinstance(pkg.sum_type, str)
```

Comment:
Is this type unknown?  In other words can it sometimes be a string and sometimes not a string?  That is not ideal, we should avoid that if possible.

Reply:
as serializer is used only in upload (and w/GET) now we will assume it must be only string.